RelhaxModpack/RelhaxModpack/Common/Translations.cs

```diff
@@ -3335,7 +3336,8 @@ public static void LoadTranslations()
             // Many users download the application right from the website and run it in the downloads folder. This is not reccomended
             English.Add("moveAppOutOfDownloads", "The application detected that it is launched from the 'Downloads' folder. This is not recommended because the application creates several folders and files " +
                 "that may be difficult to find in a large 'Downloads' folder. You should move the application and all 'Relhax' files and folders into a new folder.");
-            German.Add("moveAppOutOfDownloads", TranslationNeeded);
+            German.Add("moveAppOutOfDownloads", "Die Anwendung hat festgestellt, dass sie über den Ordner "Downloads" gestartet wurde. Dies wird nicht empfohlen, da die Anwendung mehrere Ordner und Dateien erstellt." +
```

Comment:
Remember to escape the " " or use ' ' :P I will fix it in my "mashed PR" that includes your changes :D